notebooks/openvino/phi-2_on_mtl.ipynb

```diff
@@ -0,0 +1,585 @@
+    "* `save_name`: directory where the exported & quantized model will be saved.\n",
+    "* `precision`: the compute data type we will use for inference of the model, can be either `f32` or `f16`. We use FP32 precision due to Phi-2 overflow issues in FP16.\n",
+    "* `quantization_config`: here we set the attributes for the weight only quantization algorithm:\n",
+    "    * `bits`: number of bits to use for quantization, can be either `8` or `4`.\n",
+    "    * `sym`: whether to use symmetric quantization or not, can be either `True` or `False`.\n",
+    "    * `group_size`: number of weights to group together for quantization. We use groups of 128 to ensure no accuracy degradation.\n",
+    "    * `ratio`: the ratio of the model to quantize to #`bits`. The rest will be quantize to the default bits number, `8`.\n",
+    "* `device`:  the device to use for inference, can be either `cpu` or `gpu`.\n",
+    "* `stateful`: Optimize model by setting the KV cache as part of the models state instead of as an input\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "800cd7a3-a21d-4a0a-9d73-2a2d08646f99",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = 'microsoft/phi-2'\n",
+    "save_name = './phi-2-woq4'\n",
+    "precision = 'f32'\n",
+    "quantization_config = OVWeightQuantizationConfig(\n",
+    "    bits=4,\n",
+    "    sym=False,\n",
+    "    group_size=128,\n",
+    "    ratio=0.8,\n",
+    ")\n",
+    "device = 'gpu'\n",
+    "stateful = True "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f398868-93d7-4c2d-9591-9bac8e9b701c",
+   "metadata": {},
+   "source": [
+    "With this configuration we expect the model size to reduce to around to 1.62GB:  $0.8 \\times 2.7{\\times}10^3 \\times \\frac{1}{2}\\text{B} + 0.2 * 2.7{\\times}10^3 \\times 1\\text{B} = 1.62{\\times}10^3\\text{B} = 1.62\\text{GB}$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d994997d-344c-4d6c-ab08-f78ecb7f56ec",
+   "metadata": {},
+   "source": [
+    "## Export & quantize\n",
+    "OpenVINO together with 🤗 Optimum enables you to load, export and quantize a model in a single `from_pretrained` call making the process as simple as possible.\n",
+    "Then, we will save the exported & quantized model locally on our laptop. If the model was already exported and saved before we will load the locally saved model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03a308c6-27e7-4926-8ac4-4fa0c1ca68d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load kwargs\n",
+    "load_kwargs = {\n",
+    "    'device': device,\n",
+    "    'ov_config': {\n",
+    "        \"PERFORMANCE_HINT\": \"LATENCY\",\n",
+    "        \"INFERENCE_PRECISION_HINT\": precision,\n",
+    "        \"CACHE_DIR\": os.path.join(save_name, \"model_cache\"),  # OpenVINO will use this directory as cache\n",
```

Comment:
These settings should be good by default. What is the reason for setting them here? When settings are set to a specific value users will wonder "when should I use other settings", "what do these settings mean", "what other values can I choose" but that's not explained in the notebook (and would be a distraction to explain too from my perspective)

Reply:
We don't use the precision default, last time I checked the code, if I override the ov_config to change 1 of the defaults, the other defaults do not get updated. Please check if the default `PERFORMANCE_HINT` and `CACHE_DIR` still apply while I change the default `INFERENCE_PRECISION_HINT`. Thanks

Reply:
Aah that is true.  We should fix that in optimum-intel and not completely override the config but  add the default CACHE_DIR and PERFORMANCE_HINT if they are not given. I will fix that. So yes, keep for now, maybe the notebook can be updated when we improve the default experience.